README.md

```diff
@@ -22,7 +21,8 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+var results = AuthorModel.findOne({name: "Bob"});
+console.log(results);
```

Comment:
Need to wrap `console.log` in a callback or promise since Mongoose queries are async.